src/main/resources/processes/IEDET/IEDET_REGISTRATION.bpmn

```diff
@@ -1,492 +1,113 @@
-      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${validCorrespondents == true}</bpmn:conditionExpression>
-    </bpmn:sequenceFlow>
-    <bpmn:serviceTask id="Screen_InvalidCorrespondents" name="Invalid Correspondents" camunda:expression="IEDET_INVALID_CORRESPONDENTS" camunda:resultVariable="screen">
-      <bpmn:incoming>SequenceFlow_0gae7tm</bpmn:incoming>
-      <bpmn:incoming>SequenceFlow_0qzrzfe</bpmn:incoming>
-      <bpmn:outgoing>SequenceFlow_0s727qp</bpmn:outgoing>
-    </bpmn:serviceTask>
-    <bpmn:userTask id="Validate_InvalidCorrespondents" name="Validate User Input">
-      <bpmn:incoming>SequenceFlow_0s727qp</bpmn:incoming>
-      <bpmn:outgoing>SequenceFlow_0eukthn</bpmn:outgoing>
-    </bpmn:userTask>
-    <bpmn:exclusiveGateway id="ExclusiveGateway_050t4qg">
-      <bpmn:incoming>SequenceFlow_0eukthn</bpmn:incoming>
-      <bpmn:outgoing>SequenceFlow_0gae7tm</bpmn:outgoing>
-      <bpmn:outgoing>SequenceFlow_0l9e253</bpmn:outgoing>
+    <bpmn:exclusiveGateway id="ExclusiveGateway_1f9m56y" default="Flow_0hgw4of">
+      <bpmn:incoming>Flow_1b9ma4e</bpmn:incoming>
+      <bpmn:outgoing>Flow_0hgw4of</bpmn:outgoing>
+      <bpmn:outgoing>Flow_0oasqm2</bpmn:outgoing>
+      <bpmn:outgoing>Flow_0ltvi5u</bpmn:outgoing>
     </bpmn:exclusiveGateway>
-    <bpmn:sequenceFlow id="SequenceFlow_0gae7tm" name="false" sourceRef="ExclusiveGateway_050t4qg" targetRef="Screen_InvalidCorrespondents">
-      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${valid == false}</bpmn:conditionExpression>
-    </bpmn:sequenceFlow>
-    <bpmn:sequenceFlow id="SequenceFlow_0s727qp" sourceRef="Screen_InvalidCorrespondents" targetRef="Validate_InvalidCorrespondents" />
-    <bpmn:sequenceFlow id="SequenceFlow_0eukthn" sourceRef="Validate_InvalidCorrespondents" targetRef="ExclusiveGateway_050t4qg" />
-    <bpmn:sequenceFlow id="SequenceFlow_0l9e253" sourceRef="ExclusiveGateway_050t4qg" targetRef="Screen_Correspondents">
-      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${valid == true}</bpmn:conditionExpression>
-    </bpmn:sequenceFlow>
-    <bpmn:sequenceFlow id="SequenceFlow_0qzrzfe" sourceRef="ExclusiveGateway_039b0ih" targetRef="Screen_InvalidCorrespondents">
-      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${validCorrespondents == false}</bpmn:conditionExpression>
-    </bpmn:sequenceFlow>
-    <bpmn:userTask id="Validate_Complainant" name="Validate Complainant">
-      <bpmn:incoming>SequenceFlow_02dyvii</bpmn:incoming>
-      <bpmn:outgoing>SequenceFlow_11zs70y</bpmn:outgoing>
+    <bpmn:callActivity id="CallActivity_CorrespondentInput" name="Correspondent Input" calledElement="COMPLAINT_CORRESPONDENT">
+      <bpmn:extensionElements>
+        <camunda:out source="DIRECTION" target="DIRECTION" />
+        <camunda:in source="CaseUUID" target="CaseUUID" />
+        <camunda:in sourceExpression="POGR_CORRESPONDENT_INVALID" target="CORRESPONDENT_INVALID_SCREEN" local="true" />
```

Comment:
This seems incorrect - we need to build the screen for IEDET. There was an already existing one we can pass in?